table: fix row border missing on first frame

Moulberry opened the pull request:
Currently, the bounds are zero on the first frame.
Therefore, the extra_rows_count will always be zero on the first frame.
This causes rendering issues for row border & row stripes.

Unfortunately, I don't know how to get the correct height on the first frame to fix the row stripes.

However, an easy fix for the row border rendering issue is to disable the table_is_filled check. The check is problematic anyways since the border affects the height of the element.

I attached a video showing the issue, the border doesn't render on the first frame and the height of the element also shifts due to the border:

https://github.com/user-attachments/assets/b154c686-cd9c-4b6b-8ba6-c79640297abc

Moulberry commented:
> Your change is not correct, now the vertical scrollbar is overflow to appear.

Yes, I noticed this too.
However, the stripe vertical scrollbar bug happens even without this change.

I took this screenshot on the current latest commit (without this pull request), and you can see the scrollbar is visible too:
<img width="1236" height="592" alt="image" src="https://github.com/user-attachments/assets/6db248dd-21f6-41c9-accc-c8d844214fc5" />

Please check the story example on the latest commit to confirm the bug isn't caused by this change

huacnlee commented:
You can try my update, now main branch not overflow to show scrollbar.

But your changes are still having.

Moulberry commented:
I removed the change that was causing it.
I also removed the code for disabling rendering the last border, it seems wrong:

Before:
<img width="579" height="223" alt="image" src="https://github.com/user-attachments/assets/2ac02fa9-f0f4-4112-b330-cd3f8ddafc89" />
After:
<img width="615" height="332" alt="image" src="https://github.com/user-attachments/assets/b9dd324b-c828-4e33-bf41-d404d1648c5a" />

huacnlee commented:
We need to avoid render last row bottom border.

<img width="425" height="137" alt="image" src="https://github.com/user-attachments/assets/9f95cbd8-c598-414d-8ac1-56305bce908d" />

Moulberry commented:
> We need to avoid render last row bottom border.

Ah, I understand the purpose now.

I have added a commit which makes it so the border isn't rendered when the table is full, but renders the border otherwise.

Is that correct?

huacnlee commented:
Thanks
